getting-started/templates/systemlink-values.yaml

```diff
@@ -306,6 +306,26 @@ minio:
 ## Configuration for test result storage.
 ##
 testmonitorservice:
+  ## Configuring sidecars for the Test Monitor Service.
+  ## @param sidecars Additional containers to run in the same pod as the Test Monitor Service.
+  ## It can be used for authentication proxy containers, such as the Cloud SQL Auth Proxy.
```

Comment:
Update this similar to [this](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullRequest/850970#1733928177) in all places